.circleci/config.yml

```diff
@@ -486,6 +491,26 @@ jobs:
           file: ./merged.xml
           flags: phpunit
 
+  pa11y-ci:
+    machine:
+      image: circleci/classic:latest
+    steps:
+      - checkout
+      - attach_workspace: { at: ~/project }
+      - run:
+          name: Set environment
+          command: |
+            ~/project/.circleci/set_env.sh >> $BASH_ENV
+      - run:
+          name: Run pa11y
+          command: |
+            docker-compose -f docker-compose.yml up -d pa11y
+            docker-compose exec frontend touch /var/www/.enableProdMode
+            docker-compose exec admin touch /var/www/.enableProdMode
+            docker-compose exec api touch /var/www/.enableProdMode
+            sleep 10
+            docker-compose -f docker-compose.yml run pa11y pa11y-ci || echo "Pa11y found some errors"
```

Comment:
I think it's worth adding a comment here to say what's going on with the double pipe for our future colleague's sake.

Reply:
Added a comment to explain what the purpose of the double pipe is